Cargo.toml

```
@@ -39,6 +39,11 @@ lru = "0.7.1"
 hashlink = "0.7.0"
 delay_map = "0.3.0"
 more-asserts = "0.2.2"
+async-trait = "0.1.67"
+tokio-context = "0.1.3"
+mio = "0.8.6"
+parse-display-derive = "0.8.0"
+thiserror = "1.0.40"
```

```suggestion
async-trait = "0.1.67"
parse-display-derive = "0.8.0"
thiserror = "1.0.40"
```

It doesn't seem like tokio-context and mio are used in this PR. If that is the case we probably shouldn't add the dependencies

---

Thanks kiddo!